### PR TITLE
version number entry detailed in IANA section

### LINKED ISSUE #118: Add a new IANA registry to track MP-DCCP versions

### PR DESCRIPTION
**Pull request by dirkhugo**
closes #118

### CODE REVIEW

**Comment by dirkhugo** [comment]
@boucadair : Does this address your comment, Med?

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -1340,6 +1340,8 @@ added to the "Feature Numbers Registry" according to {{RFC4340}}, Section 19.4.
 | 0x10  | MP-DCCP capability feature | [ThisDocument]  |
 {:#ref-add-feature-list title='Addition to DCCP Feature list Entries'}
 
+As outlined in sect. {{mp_capable}} the new 1-Byte entry above includes a 4-bit part to specify the version of the used MP-DCCP implementation.
```
Comment (boucadair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:1343]:
```suggestion
As outlined in sect. {{mp_capable}} the new 1-Byte entry above includes a 4-bit part to specify the version of the used MP-DCCP implementation. This document requests IANA to create a new sub-registry under the MP-DCCP registry to track the MP-DCCP version. The initial content of this sub-registry is as follows: 

<<<Table with the value that is assigned in this document + list of values for future assignments>>>
```

**Review by dirkhugo — COMMENTED** [review]
@boucadair : Addresses this text and format your comments? Thanks for clarification!

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 2]
```diff
@@ -1340,6 +1340,17 @@ added to the "Feature Numbers Registry" according to {{RFC4340}}, Section 19.4.
...
+As outlined in sect. {{mp_capable}} the new 1-Byte entry above includes a 4-bit part to specify the version of the used MP-DCCP implementation. This document requests IANA to create a new sub-registry under the MP-DCCP registry to track the MP-DCCP version. The initial content of this sub-registry is as follows: 
+
+
+   |  Version  | Unassigned | Specification  |
+   |:---------:|:----------:|:--------------:|
+   |   0000    |   0000     | [ThisDocument] |
+   |0001 - 1111|   0000     |      TBD       |
+{:#ref-add-version-list title='Sub-registry for tracking the MP-DCCP version'}
+
+Future MP-DCCP versions 1 to 15 are assigned in this sub-registry using the Specification Required policy (Section 4.6 of {{RFC8126}}).
```
Comment (boucadair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:1352]:
```suggestion
Future MP-DCCP versions 1 to 15 are assigned from this sub-registry using the Specification Required policy (Section 4.6 of {{RFC8126}}).
```

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 3]
```diff
@@ -1340,6 +1340,17 @@ added to the "Feature Numbers Registry" according to {{RFC4340}}, Section 19.4.
 | 0x10  | MP-DCCP capability feature | [ThisDocument]  |
 {:#ref-add-feature-list title='Addition to DCCP Feature list Entries'}
 
+As outlined in sect. {{mp_capable}} the new 1-Byte entry above includes a 4-bit part to specify the version of the used MP-DCCP implementation. This document requests IANA to create a new sub-registry under the MP-DCCP registry to track the MP-DCCP version. The initial content of this sub-registry is as follows: 
+
+
+   |  Version  | Unassigned | Specification  |
```
Comment (boucadair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:1346]:
```suggestion
   |  Version  | Description | Specification  |
```

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 4]
```diff
@@ -1340,6 +1340,17 @@ added to the "Feature Numbers Registry" according to {{RFC4340}}, Section 19.4.
...
+   |:---------:|:----------:|:--------------:|
+   |   0000    |   0000     | [ThisDocument] |
```
Comment (boucadair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:1348]:
```suggestion
   |   0000    |   Version 0| [ThisDocument] |
```

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 5]
```diff
@@ -1340,6 +1340,17 @@ added to the "Feature Numbers Registry" according to {{RFC4340}}, Section 19.4.
...
+   |0001 - 1111|   0000     |      TBD       |
```
Comment (boucadair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:1349]:
```suggestion
   |0001 - 1111|   Unassigned     |                 |
```

**Review by boucadair — CHANGES_REQUESTED** [review]

**Comment by dirkhugo** [comment]
so we can close this issue, right? merci!

**Comment by boucadair** [comment]
> so we can close this issue, right? merci!

I think so, thanks.

**Pull request merged**